keyform/views.py

```diff
@@ -37,6 +38,15 @@ def form_valid(self, form):
             return self.form_invalid(form)
 
     def get_form(self, form_class=None):
-        form = CreateForm(**self.get_form_kwargs())
+        form = CreateForm(instance=Request(staff=get_user_model().objects.all()[0]), **self.get_form_kwargs())
```

Comment:
This code is simply picking the first user instead of the currently logged in user.